app/src/main/res/raw/empty_state.json

```diff
@@ -0,0 +1 @@
0.886,0],[0,0],[-0.455,0.541],[0,0],[0,1.657],[0,0],[0,0],[0,1.658],[0,0],[1.657,0],[0,0],[0,0],[1.657,0],[0,0],[0,-1.657]],"v":[[44.716,7.857],[34.871,7.857],[34.871,-52.047],[31.871,-55.047],[7.087,-55.047],[4.79,-53.978],[-47.012,7.598],[-47.716,9.53],[-47.716,32.919],[-44.716,35.919],[4.087,35.919],[4.087,52.047],[7.087,55.047],[31.871,55.047],[34.871,52.047],[34.871,35.919],[44.716,35.919],[47.716,32.919],[47.716,10.857]],"c":true},"ix":2},"nm":"Path 2","mn":"ADBE Vector Shape - Group","hd":false},{"ty":"mm","mm":1,"nm":"Merge Paths 1","mn":"ADBE Vector Filter - Merge","hd":false},{"ty":"fl","c":{"a":0,"k":[0,0.3764705882352941,0.39215686274509803,1],"ix":4},"o":{"a":0,"k":100,"ix":5},"r":1,"bm":0,"nm":"Fill 1","mn":"ADBE Vector Graphic - Fill","hd":false},{"ty":"tr","p":{"a":0,"k":[47.966,55.297],"ix":2},"a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[100,100],"ix":3},"r":{"a":0,"k":0,"ix":6},"o":{"a":0,"k":100,"ix":7},"sk":{"a":0,"k":0,"ix":4},"sa":{"a":0,"k":0,"ix":5},"nm":"Transform"}],"nm":"Group 1","np":4,"cix":2,"bm":0,"ix":1,"mn":"ADBE Vector Group","hd":false}],"ip":0,"op":60.0000024438501,"st":0,"bm":0},{"ddd":0,"ind":6,"ty":4,"nm":"Layer 7 Outlines","sr":1,"ks":{"o":{"a":0,"k":100,"ix":11},"r":{"a":0,"k":0,"ix":10},"p":{"a":0,"k":[64.633,99.127,0],"ix":2},"a":{"a":0,"k":[47.966,55.297,0],"ix":1},"s":{"a":0,"k":[100,100,100],"ix":6}},"ao":0,"shapes":[{"ty":"gr","it":[{"ind":0,"ty":"sh","ix":1,"ks":{"a":0,"k":{"i":[[0,0],[0,0],[0,0]],"o":[[0,0],[0,0],[0,0]],"v":[[4.087,7.857],[-13.799,7.857],[4.087,-13.19]],"c":true},"ix":2},"nm":"Path 1","mn":"ADBE Vector Shape - Group","hd":false},{"ind":1,"ty":"sh","ix":2,"ks":{"a":0,"k":{"i":[[1.657,0],[0,0],[0,0],[1.657,0],[0,0],[0.57,-0.677],[0,0],[0,-0.707],[0,0],[-1.657,0],[0,0],[0,0],[-1.657,0],[0,0],[0,1.658],[0,0],[0,0],[0,1.657],[0,0]],"o":[[0,0],[0,0],[0,-1.657],[0,0],[-0.886,0],[0,0],[-0.455,0.541],[0,0],[0,1.657],[0,0],[0,0],[0,1.658],[0,0],[1.657,0],[0,0],[0,0],[1.657,0],[0,0],[0,-1.657]],"v":[[44.717,7.857],[34.87,7.857],[34.87,-52.047],[31.87,-55.047],[7.087,-55.047],[4.79,-53.978],[-47.012,7.598],[-47.717,9.53],[-47.717,32.919],[-44.717,35.919],[4.087,35.919],[4.087,52.047],[7.087,55.047],[31.87,55.047],[34.87,52.047],[34.87,35.919],[44.717,35.919],[47.717,32.919],[47.717,10.857]],"c":true},"ix":2},"nm":"Path 2","mn":"ADBE Vector Shape - Group","hd":false},{"ty":"mm","mm":1,"nm":"Merge Paths 1","mn":"ADBE Vector Filter - Merge","hd":false},{"ty":"fl","c":{"a":0,"k":[0,0.3764705882352941,0.39215686274509803,1],"ix":4},"o":{"a":0,"k":100,"ix":5},"r":1,"bm":0,"nm":"Fill 1","mn":"ADBE Vector Graphic - Fill","hd":false},{"ty":"tr","p":{"a":0,"k":[47.966,55.297],"ix":2},"a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[100,100],"ix":3},"r":{"a":0,"k":0,"ix":6},"o":{"a":0,"k":100,"ix":7},"sk":{"a":0,"k":0,"ix":4},"sa":{"a":0,"k":0,"ix":5},"nm":"Transform"}],"nm":"Group 1","np":4,"cix":2,"bm":0,"ix":1,"mn":"ADBE Vector Group","hd":false}],"ip":0,"op":60.0000024438501,"st":0,"bm":0}],"markers":[]}
```

Comment:
add an empty line here to avoid this error

Reply:
Done